gensim/models/hdpmodel.py

```diff
@@ -612,10 +604,9 @@ def show_topic_terms(self, topic_data, num_words):
     def format_topic(self, topic_id, topic_terms):
         if self.STYLE_GENSIM == self.style:
             fmt = ' + '.join(['%.3f*%s' % (weight, word) for (word, weight) in topic_terms])
-            fmt = 'topic %i: %s' % (topic_id, fmt)
         else:
             fmt = '\n'.join(['    %20s    %.8f' % (word, weight) for (word, weight) in topic_terms])
-            fmt = 'topic %i:\n%s' % (topic_id, fmt)
```

Comment:
but the Travis build will fail. I have added all the other changes but cannot commit until the tests pass on my local machine.